versions/1/kibana/spec.yml

```diff
@@ -65,3 +65,12 @@
         type: file
         contentMediaType: "application/json"
         pattern: '^.+\.json$'
+    - description: Folder containing rules
+      type: folder
+      name: "security_rule"
+      required: false
+      contents:
+      - description: An individual rule file for the detection engine
+        type: file
+        contentMediaType: "application/json"
+        pattern: '^rule-.+\.json$'
```

Comment:
Is the `rule` prefix in the name required?

Reply:
no, I can loose this to just `*.json`

Reply:
Just `^.+\.json$` as of ba237067d01321af83c722ce03fb9fb93a312a81